examples/istio/main.tf

```diff
@@ -2,32 +2,31 @@ provider "aws" {
   region = local.region
 }
 
-provider "kubernetes" {
+provider "kubectl" {
   host                   = module.eks.cluster_endpoint
   cluster_ca_certificate = base64decode(module.eks.cluster_certificate_authority_data)
+  token                  = data.aws_eks_cluster_auth.this.token
+  load_config_file       = false
+}
 
-  exec {
```

Comment:
the exec method is the method that is recommended by the Hashicorp providers (Kubernetes, Helm) - lets not change this